[2022-05-11 22:05:00,271] logger_util.py->print_log line:14 [INFO] ----------接口测试开始----------
[2022-05-11 22:05:00,272] logger_util.py->print_log line:14 [INFO] 接口名称：成功获取token鉴权码
[2022-05-11 22:05:00,272] logger_util.py->print_log line:14 [INFO] 请求方式：get
[2022-05-11 22:05:00,272] logger_util.py->print_log line:14 [INFO] 请求路径：https://api.weixin.qq.com/cgi-bin/token
[2022-05-11 22:05:00,272] logger_util.py->print_log line:14 [INFO] 请求headers参数：None
[2022-05-11 22:05:00,273] logger_util.py->print_log line:14 [INFO] 请求params参数：{'grant_type': 'client_credential', 'appid': 'wx8a9de038e93f77ab', 'secret': '8326fc915928dee3165720c910effb86'}
[2022-05-11 22:05:00,581] logger_util.py->print_log line:14 [INFO] 预期结果：[{'codes': {'status_code': 200}}, {'contains': 'access_token'}]
[2022-05-11 22:05:00,581] logger_util.py->print_log line:14 [INFO] 实际结果：{'access_token': '56_-H_-ofNvcW2pAllLlo-8xX22W8SjDI5PTLJ3WGYAvjQ0e0fdFeChkQIm6UJXLwBx_C0UPJZb4j_psbtzRRhNmhoDlj6Ml24oU9N3lpLMxDwNIYDlbyvAjmcN13XLrTYyLyLY2yueCPzkBPrGISBiAHAPZI', 'expires_in': 7200}
[2022-05-11 22:05:00,581] logger_util.py->print_log line:14 [INFO] 接口测试成功
[2022-05-11 22:05:00,581] logger_util.py->print_log line:14 [INFO] ----------接口测试结束----------

[2022-05-11 22:05:00,583] logger_util.py->print_log line:14 [INFO] ----------接口测试开始----------
[2022-05-11 22:05:00,584] logger_util.py->print_log line:14 [INFO] 接口名称：检查appid为空
[2022-05-11 22:05:00,584] logger_util.py->print_log line:14 [INFO] 请求方式：get
[2022-05-11 22:05:00,584] logger_util.py->print_log line:14 [INFO] 请求路径：https://api.weixin.qq.com/cgi-bin/token
[2022-05-11 22:05:00,584] logger_util.py->print_log line:14 [INFO] 请求headers参数：None
[2022-05-11 22:05:00,585] logger_util.py->print_log line:14 [INFO] 请求params参数：{'grant_type': 'client_credential', 'appid': 100, 'secret': '8326fc915928dee3165720c910effb86'}
[2022-05-11 22:05:00,622] logger_util.py->print_log line:14 [INFO] 预期结果：[{'codes': {'status_code': 200}}, {'contains': 41002}]
[2022-05-11 22:05:00,622] logger_util.py->print_log line:14 [INFO] 实际结果：{'errcode': 40013, 'errmsg': 'invalid appid rid: 627bc28b-613aabd4-363b77e3'}
[2022-05-11 22:05:00,622] logger_util.py->error_log line:10 [ERROR] contains断言失败：返回结果中没有41002
[2022-05-11 22:05:00,623] logger_util.py->error_log line:10 [ERROR] 断言报错：Traceback (most recent call last):
  File "D:\workspace\pythonwork\vip_pytest_one\commons\assert_utils.py", line 22, in assert_result
    flag = contains_assert(value, sj_result)
  File "D:\workspace\pythonwork\vip_pytest_one\commons\assert_utils.py", line 75, in contains_assert
    error_log("contains断言失败：返回结果中没有"+str(value)+"")
  File "D:\workspace\pythonwork\vip_pytest_one\commons\logger_util.py", line 11, in error_log
    raise Exception(message)
Exception: contains断言失败：返回结果中没有41002

[2022-05-11 22:05:00,623] logger_util.py->error_log line:10 [ERROR] 规范YAML测试用例报错：断言报错：Traceback (most recent call last):
  File "D:\workspace\pythonwork\vip_pytest_one\commons\assert_utils.py", line 22, in assert_result
    flag = contains_assert(value, sj_result)
  File "D:\workspace\pythonwork\vip_pytest_one\commons\assert_utils.py", line 75, in contains_assert
    error_log("contains断言失败：返回结果中没有"+str(value)+"")
  File "D:\workspace\pythonwork\vip_pytest_one\commons\logger_util.py", line 11, in error_log
    raise Exception(message)
Exception: contains断言失败：返回结果中没有41002

[2022-05-11 22:05:00,767] logger_util.py->print_log line:14 [INFO] ----------接口测试开始----------
[2022-05-11 22:05:00,767] logger_util.py->print_log line:14 [INFO] 接口名称：检查secret为空1
[2022-05-11 22:05:00,767] logger_util.py->print_log line:14 [INFO] 请求方式：get
[2022-05-11 22:05:00,768] logger_util.py->print_log line:14 [INFO] 请求路径：https://api.weixin.qq.com/cgi-bin/token
[2022-05-11 22:05:00,768] logger_util.py->print_log line:14 [INFO] 请求headers参数：None
[2022-05-11 22:05:00,768] logger_util.py->print_log line:14 [INFO] 请求params参数：{'grant_type': 'client_credential', 'appid': 'wx8a9de038e93f77ab', 'secret': ''}
[2022-05-11 22:05:00,891] logger_util.py->print_log line:14 [INFO] 预期结果：[{'codes': {'status_code': 200}}, {'contains': 41004}]
[2022-05-11 22:05:00,891] logger_util.py->print_log line:14 [INFO] 实际结果：{'errcode': 41004, 'errmsg': 'appsecret missing rid: 627bc28c-5a35fd08-43db2c26'}
[2022-05-11 22:05:00,891] logger_util.py->print_log line:14 [INFO] 接口测试成功
[2022-05-11 22:05:00,891] logger_util.py->print_log line:14 [INFO] ----------接口测试结束----------

[2022-05-11 22:05:00,894] logger_util.py->print_log line:14 [INFO] ----------接口测试开始----------
[2022-05-11 22:05:00,894] logger_util.py->print_log line:14 [INFO] 接口名称：查询标签接口
[2022-05-11 22:05:00,896] logger_util.py->error_log line:10 [ERROR] request下面必须包含有method,base_url,url这三个二级关键字
[2022-05-11 22:05:00,896] logger_util.py->error_log line:10 [ERROR] 规范YAML测试用例报错：request下面必须包含有method,base_url,url这三个二级关键字
[2022-05-11 22:05:00,908] logger_util.py->print_log line:14 [INFO] ----------接口测试开始----------
[2022-05-11 22:05:00,908] logger_util.py->print_log line:14 [INFO] 接口名称：文件上传接口
[2022-05-11 22:05:00,909] logger_util.py->print_log line:14 [INFO] 请求方式：post
[2022-05-11 22:05:00,909] logger_util.py->print_log line:14 [INFO] 请求路径：https://api.weixin.qq.com/cgi-bin/media/uploadimg
[2022-05-11 22:05:00,909] logger_util.py->print_log line:14 [INFO] 请求headers参数：None
[2022-05-11 22:05:00,910] logger_util.py->print_log line:14 [INFO] 请求params参数：{'access_token': '56_-H_-ofNvcW2pAllLlo-8xX22W8SjDI5PTLJ3WGYAvjQ0e0fdFeChkQIm6UJXLwBx_C0UPJZb4j_psbtzRRhNmhoDlj6Ml24oU9N3lpLMxDwNIYDlbyvAjmcN13XLrTYyLyLY2yueCPzkBPrGISBiAHAPZI'}
[2022-05-11 22:05:01,490] logger_util.py->print_log line:14 [INFO] 预期结果：None
[2022-05-11 22:05:01,491] logger_util.py->print_log line:14 [INFO] 实际结果：{'url': 'http://mmbiz.qpic.cn/mmbiz_jpg/gwVWn1TZHhUL50TZaUjDkImand8Ol8IcezeuyPic9vCKwbiarqgzmMxqrXJCdciaqfECWicabVQTX2O1SWWiaHAhunQ/0'}
[2022-05-11 22:05:01,491] logger_util.py->print_log line:14 [INFO] 接口测试成功
[2022-05-11 22:05:01,491] logger_util.py->print_log line:14 [INFO] ----------接口测试结束----------

